cmd/ebrelayer/relayer/cosmos.go

```diff
@@ -372,8 +374,33 @@ func getOracleClaimType(eventType string) types.Event {
 	return claimType
 }
 
+func tryInitRelayConfig(sub CosmosSub, claimType types.Event) (*ethclient.Client, *bind.TransactOpts, common.Address, error) {
```

Comment:
would 'retryInitRelayConfig' be a slightly better method name here?

Reply:
Adding Re-try implies that this method is only used for retrying, which is misleading as it is used for the initialization. If that fails, then this method is used for retrying.